runtime/util/module.tdf

```diff
@@ -55,5 +55,5 @@ TraceEvent=Trc_MODULE_addModulePackage Overhead=1 Level=5 Test Template="JVM_Add
 TraceEvent=Trc_MODULE_setPackage Overhead=1 Level=5 Test Template="internally set package/class: %.*s, loader: %s (0x%p), module: %s (0x%p)"
 TraceEvent=Trc_MODULE_invokeHashTableAtPut Overhead=1 Level=5 Test Template="%s invokes hashTableAtPut() : 0x%p->table (0x%p), value (0x%p, 0x%p) collisionIsFailure (%s)"
 TraceEvent=Trc_MODULE_hashTableAtPut NoEnv Overhead=1 Level=5 Test Template="hashTableAtPut() : table (0x%p), value (0x%p) node (0x%p) collisionIsFailure(false)"
-TraceEvent=Trc_MODULE_setUnamedModuleForSystemLoaderModuleObject Overhead=1 Level=5 Template="JVM_SetBootLoaderUnnamedModule() sets unnamed module (0x%p) for unamedModuleForSystemLoader (0x%p)"
+TraceEvent=Trc_MODULE_setUnnamedModuleForSystemLoaderModuleObject Overhead=1 Level=5 Template="JVM_SetBootLoaderUnnamedModule() sets unnamed module (0x%p) for unnamedModuleForSystemLoader (0x%p)"
```

Comment:
@pshipton Do we need to obsolete the tracepoint and add a new one since the name is changed?

Reply:
Hmm, I don't think so. We don't want to change the tracepoint numbers since some are hard coded in tests, healthcenter, maybe doc, but I don't think changing the name is a problem.